data.json

```diff
@@ -173,11 +173,11 @@
   {
     "name": "Magento",
```

Comment:
Can you change for either Magento + Adobe or the other way around?

Reply:
Updated accordingly